cmd/difference.go

```diff
@@ -82,7 +82,7 @@ func getSourceModTimeKey(metadata map[string]string) string {
 
 // activeActiveModTimeUpdated tries to calculate if the object copy in the target
 // is older than the one in the source by comparing the modtime of the data.
-func activeActiveModTimeUpdated(src, dst *ClientContent) bool {
+func activeActiveModTimeUpdated(src, dst *ClientContent, isActiveActive bool) bool {
```

Comment:
Should this function be called at all in an active-active mirror setting?

Reply:
I guess so. 
This part of the code was added/modified by @harshavardhana.
We can consult with him.